docs/learn/learn-identity.md

```diff
@@ -60,18 +60,19 @@ There are multiple registrars on {{ polkadot: Polkadot :polkadot }}{{ kusama: Ku
 Unless no additional information is available here, you must reach out to specific registrars
 individually if you want to be judged by those.
 
-:::info Registrar 0 decommission on April 1st, 2024
+:::info Registrar 0 decommissioned on April 1st, 2024
 
 From the 1st of April 2024 onwards, Registrar 0 will still exist on-chain but will not accept any
-new judgment requests. Identities judged by the registrar before that date will not be affected. For
-new identity judgment, use the other registrars.
+new judgment requests. The registrar fees is set to a substantial amount to dissuade identity judgement
```

Comment:
```suggestion
new judgment requests. The registrar fees are set to a substantial amount to dissuade identity judgment
```

Reply:
The section title itself is "Judgement" :)

The Wiki doc stuck to UK english with this word.